components/stmn/Create/CreateToken.vue

```diff
@@ -0,0 +1,269 @@
+<template>
+  <div>
+    <Loader v-model="isLoading" :status="status" />
+    <BaseTokenForm
+      v-bind.sync="base"
+      ref="baseTokenForm"
+      :collections="collections"
+      :show-explainer-text="showExplainerText">
+      <template #main>
+        <BasicSwitch key="nsfw" v-model="nsfw" label="mint.nfsw" />
+        <BasicSwitch key="listed" v-model="listed" label="mint.listForSale" />
+
+        <BalanceInput
+          v-if="listed"
+          ref="balanceInput"
+          key="price"
+          label="Price"
+          expanded
+          required
+          :has-to-larger-than-zero="listed"
+          :step="0.1"
+          class="mb-3"
+          @input="updatePrice" />
+
+        <div v-show="base.selectedCollection" key="attributes">
+          <CustomAttributeInput
+            v-model="tags"
+            :max="10"
+            class="mb-3"
+            visible="collapse.collection.attributes.show"
+            hidden="collapse.collection.attributes.hide" />
+        </div>
+      </template>
+      <template #footer>
+        <NeoField key="advanced">
+          <CollapseWrapper
+            v-if="base.copies > 1"
+            visible="mint.expert.show"
+            hidden="mint.expert.hide"
+            class="mt-3">
+            <BasicSwitch
+              v-model="postfix"
+              class="mt-3"
+              label="mint.expert.postfix" />
+          </CollapseWrapper>
+        </NeoField>
+        <NeoField
+          v-if="isLogIn"
+          key="submit"
+          variant="danger"
+          :message="balanceNotEnoughMessage">
+          <SubmitButton
+            expanded
+            label="mint.submit"
+            :loading="isLoading"
+            @click="submit()" />
+        </NeoField>
+      </template>
+    </BaseTokenForm>
+  </div>
+</template>
+
+<script lang="ts">
```

Comment:
can't this be written using vue composition api?

Reply:
it is already a big PR
let's do this here:
* #6336

Reply:
Hi @roiLeo 
I ended up converting it to composition API in this PR

along the way I've edited useGraphql to add the ability to call refetch with new parameters